.github/workflows/release-push-image.yaml

```
@@ -38,14 +40,6 @@ jobs:
       java-distribution: ${{ inputs.java-distribution || 'temurin' }}
       java-version: ${{ inputs.java-version || '21.0.6' }}
 
-  check-docker:
-    name: Docker
-    uses: ./.github/workflows/zxc-verify-docker-build-determinism.yaml
-    with:
-      ref: ${{ github.event.inputs.ref || '' }}
-      java-distribution: ${{ inputs.java-distribution || 'temurin' }}
-      java-version: ${{ inputs.java-version || '21.0.6' }}
-
   publish:
     needs: [check-gradle, check-docker]
```

`check-docker` should be removed.
```suggestion
    needs: [check-gradle]
```

---

Thanks.